Amasty_Storelocator/styles/modules/_store-locator.scss

```diff
@@ -142,6 +142,11 @@
             &.focus-visible {
                 background-color: $store-locator__button-bg--active;
             }
+
+            &:disabled,
+            &[disabled] {
+               @extend .button:disabled;
```

Comment:
wrong indentation ` 148:16  error  Expected indentation of 16 spaces but found 15 `
it should also fix lints